packs/data/age-of-ashes-bestiary.db/alak-stagram.json

```diff
@@ -35,6 +35,7 @@
                     "value": "<p>This thin spear is well balanced for throwing but is not designed for melee use.</p>"
                 },
                 "equipped": {
+                    "-=inSlot": null,
```

Comment:
This looks like the -= bug...

Reply:
Could be, happens on all equipment with the refresh button

Reply:
Removed